src/main/java/com/mentalhealthapi/mentalhealthapi/dao/BlogServiceStub.java

```diff
@@ -1,4 +1,4 @@
-package com.mentalhealthapi.mentalhealthapi.service;
+package com.mentalhealthapi.mentalhealthapi.dao;
```

Comment:
It looks like this is a service class, so it should be in a service package.  I'd leave this as it were.